ddspipe_participants/src/cpp/reader/rtps/CommonReader.cpp

```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <fastrtps/rtps/RTPSDomain.h>
-#include <fastrtps/rtps/participant/RTPSParticipant.h>
+#include <fastdds/rtps/RTPSDomain.h>
```

Comment:
There are different styles when it comes to this. I would, because you're initializing an instance of `ReaderQos` in the source. I only don't include them when they only appear in function's signatures, because then you're forced by C++ to include them in the header, so you can be sure it's still included.